spec/spec_helper_local.rb

```diff
@@ -36,7 +36,11 @@ def with_openbsd_facts
 def with_redhat_facts
   let :facts do
     super().merge(operatingsystemmajrelease: '7',
```

Comment:
nitpicking, but could you add a newline after the ( in line 38 and before the { in line 40?